src/demo/ShowcaseItem/ShowcaseItem.tsx

```diff
@@ -5,7 +5,7 @@ import {cn} from '../../components/utils/cn';
 import './ShowcaseItem.scss';
 
 interface ShowcaseItemProps {
-    title: string;
+    title: React.ReactNode;
```

Comment:
What we want here except strings?

Reply:
reverted https://github.com/gravity-ui/uikit/pull/1597/commits/0f204a4f7a4f62b80c66b0d62fa9765b588a4d9a